modules/starting-a-basic-workbench.adoc

```diff
@@ -32,7 +32,7 @@ endif::[]
 If your credentials are accepted, the *Workbench control panel* opens displaying the *Start a basic workbench* page.
 . Start a basic workbench.
 
-.. In the *Workbench image* section, select the workbench image to use for your server. 
+.. In the *Workbench image* section, select the workbench image to use for your workbench. 
```

Comment:
It uses the word workbench multiple times in close succession, which can make it feel repetitive.

Reply:
@erwangranger forebode using the word "server", so I believe this is how this came to be